Develop fails to compile

## Description
Looks like soca got out of sync with ufo? Here is the message i get when trying to compile on orion with GNU.

```
/home/sfrolov/work/jedi/src/soca/develop/bundle/soca/src/soca/GetValues/GetValues.cc: In member function ‘void soca::GetValues::getValuesFromFile(const ufo::Locations&, const oops::Variables&, ufo::GeoVaLs&) const’:
/home/sfrolov/work/jedi/src/soca/develop/bundle/soca/src/soca/GetValues/GetValues.cc:94:46: error: no matching function for call to ‘ioda::ObsSpace::ObsSpace(eckit::LocalConfiguration&, const eckit::mpi::Comm&, util::DateTime&, util::DateTime&, const eckit::mpi::Comm&)’
   94 |                           oops::mpi::myself());
      |                                              ^
In file included from /home/sfrolov/work/jedi/src/soca/develop/bundle/soca/src/soca/GetValues/GetValues.cc:10:
/home/sfrolov/work/jedi/build/soca/develop/ioda/include/ioda/ObsSpace.h:139:9: note: candidate: ‘ioda::ObsSpace::ObsSpace(const ioda::ObsSpace&)’
  139 |         ObsSpace(const ObsSpace &);
      |         ^~~~~~~~
/home/sfrolov/work/jedi/build/soca/develop/ioda/include/ioda/ObsSpace.h:139:9: note:   candidate expects 1 argument, 5 provided
/home/sfrolov/work/jedi/build/soca/develop/ioda/include/ioda/ObsSpace.h:136:9: note: candidate: ‘ioda::ObsSpace::ObsSpace(const Parameters_&, const eckit::mpi::Comm&, const util::DateTime&, const util::DateTime&, const eckit::mpi::Comm&)’
  136 |         ObsSpace(const Parameters_ & params, const eckit::mpi::Comm & comm,
      |         ^~~~~~~~
/home/sfrolov/work/jedi/build/soca/develop/ioda/include/ioda/ObsSpace.h:136:38: note:   no known conversion for argument 1 from ‘eckit::LocalConfiguration’ to ‘const Parameters_&’ {aka ‘const ioda::ObsTopLevelParameters&’}
  136 |         ObsSpace(const Parameters_ & params, const eckit::mpi::Comm & comm,
      |                  ~~~~~~~~~~~~~~~~~~~~^~~~~~
[ 92%] Building Fortran object soca/src/soca/CMakeFiles/soca.dir/Geometry/soca_geom_mod.F90.o
[ 92%] Building Fortran object soca/src/soca/CMakeFiles/soca.dir/Transforms/Balance/soca_ksshts_mod.F90.o
[ 92%] Building Fortran object soca/src/soca/CMakeFiles/soca.dir/Transforms/Balance/soca_kst_mod.F90.o
[ 92%] Building Fortran object soca/src/soca/CMakeFiles/soca.dir/Utils/soca_omb_stats_mod.F90.o
make[2]: *** [soca/src/soca/CMakeFiles/soca.dir/GetValues/GetValues.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [soca/src/soca/CMakeFiles/soca.dir/all] Error 2
make: *** [all] Error 2
```



### To Reproduce:
Trying to build develop brunch of soca/bundle on orion. 

modules loaded
```
export JEDI_OPT=/work/noaa/da/grubin/opt/modules
module use $JEDI_OPT/modulefiles/core
module load jedi/gnu-openmpi

Currently Loaded Modules:
  1) git/2.28.0        7) zlib/1.2.11         13) hdf5/1.12.0           19) eigen/3.3.7           25) fckit/ecmwf-0.9.2
  2) git-lfs/2.11.0    8) udunits/2.2.28      14) pnetcdf/1.12.1        20) bufr/noaa-emc-11.5.0  26) atlas/ecmwf-0.24.1
  3) python/3.7.5      9) gsl_lite/0.37.0     15) netcdf/4.7.4          21) pybind11/2.7.0        27) jedi/gnu-openmpi/10.2.0
  4) gcc/10.2.0       10) lapack/3.8.0        16) nccmp/1.8.7.0         22) cmake/3.18.1
  5) jedi-gnu/10.2.0  11) openmpi/4.0.4       17) pio/2.5.1-debug       23) ecbuild/ecmwf-3.6.1
  6) szip/2.1.1       12) jedi-openmpi/4.0.4  18) boost-headers/1.73.0  24) eckit/ecmwf-1.16.0
```

--- a/.github/travisci/build.sh
+++ b/.github/travisci/build.sh
@@ -49,7 +49,7 @@ for repo in $LIB_REPOS; do
     build_opt_var=BUILD_OPT_${repo_underscore}
     build_opt="$BUILD_OPT ${!build_opt_var}"
     time ecbuild $src_dir -DCMAKE_INSTALL_PREFIX=${install_dir} -DCMAKE_BUILD_TYPE=${LIB_BUILD_TYPE} \
-            -DCMAKE_CXX_COMPILER_LAUNCHER=$CCACHE -DENABLE_TESTS=OFF -DBUILD_TESTING=OFF $build_opt
+            -DCMAKE_CXX_COMPILER_LAUNCHER=$CCACHE -DBUILD_TESTING=OFF $build_opt
 
     # build and install
     time make -j4

--- a/README.md
+++ b/README.md
@@ -26,7 +26,7 @@ make -j 4
 3. If building the same way travis-ci builds the MOM6 SOCA bundle:
 ```
 export MAIN_REPO=soca
-export LIB_REPOS="fms gsw mom6 crtm fckit atlas oops saber ioda ufo ioda-converters"
+export LIB_REPOS="jedicmake fms gsw mom6 crtm oops saber ioda ufo"
 export BUILD_OPT=""
 export BUILD_OPT_crtm="-DBUILD_CRTM=ON"
 export BUILD_OPT_oops="-DENABLE_QG_MODEL=OFF -DENABLE_LORENZ95_MODEL=OFF"

